ui/src/main/resources/Meeting/Code/Translations.xml

```diff
@@ -36,7 +36,10 @@
   <minorEdit>false</minorEdit>
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
-  <content>contrib.hr.notification.additionalPart=His/her message:
+  <content>admin.meetings=Meetings
+admin.meetings.description=Configure the homepage of your Meetings Application.
+
+contrib.hr.notification.additionalPart=His/her message:
```

Comment:
I know you didn't touch this key, but it seems to be for a different application (holiday request?). Could you check if it's used by the Meeting Application, and if not then remove it? Thanks.

Reply:
It seems to be used. Should I rename it?